test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -253,7 +254,7 @@ std::string RandomCode::rndByteSequence(int _length, SizeStrictness _sizeType)
 }
 
 //generate smart random code
-std::string RandomCode::generate(int _maxOpNumber, RandomCodeOptions _options)
+std::string RandomCode::generate(int _maxOpNumber, RandomCodeOptions& _options)
```

Comment:
Can this be `RandomCodeOptions const&`?

Reply:
hmm. although if I move opCodeProbability out of options I guess it might become const

Reply:
Which line changes `opCodeProbability`?